common/src/main/java/org/apache/uniffle/common/web/JettyServer.java

```diff
@@ -169,13 +169,15 @@ public Server getServer() {
     return this.server;
   }
 
-  public void start() throws Exception {
+  public int start() throws Exception {
     try {
       server.start();
+      httpPort = ((ServerConnector) server.getConnectors()[0]).getLocalPort();
```

Comment:
Is there anything that ensures `server.getConnectors()[0]` is an instance of the `ServerConnector` type?

Reply:
The type and sort of `ServerConnector` in determined in  `JettyServer`  initialize